symforce/codegen/template_util.py

```diff
@@ -130,11 +130,13 @@ def add_preamble(source: str, name: Path, comment_prefix: str) -> str:
 
 
 @functools.lru_cache
-def jinja_env(template_dir: T.Openable) -> RelEnvironment:
+def jinja_env(template_dir: T.Openable, search_paths: T.Iterable[T.Openable] = ()) -> RelEnvironment:
     """
     Helper function to cache the Jinja environment, which enables caching of loaded templates
     """
-    loader = jinja2.FileSystemLoader(os.fspath(template_dir))
+    all_search_paths = [os.fspath(template_dir)]
+    all_search_paths.extend(search_paths)
```

Comment:
We need to call `os.fspath` on all the entries of `search_paths` as well here (the linter may catch this, not sure)